_sass/_libs/wds/stylesheets/components/_accordions.scss

```diff
@@ -4,70 +4,99 @@ $accordion-border: 3px solid $color-gray-lightest;
 
 // Accordion Styles
 
+@mixin accordion-list-styles {
```

Comment:
I'm a bit wary of these style changes being made directly to the web design standards stylesheets. From the conversation that we had with Shawn and Julia we had decided that working with a set version of the standards was a good idea, but I'm thinking that it would be better to layer our stylesheets _on top of_ the standards instead of making changes directly to them. Does that make sense?